tools/packaging/packaging.psm1

```diff
@@ -306,7 +306,6 @@ function Start-PSPackage {
                         PackageNameSuffix = 'fxdependent'
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-macos`
Verify sub-directory creation under root.Can create a sub directory under root path
```powershell
Expected path '\TestDir-e4930d6c-2f03-4823-8c9c-c999fdef9dfa' to exist, but it did not exist.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 1467
1467:         $dirPath | Should -Exist
```